ajax/libs/timbre/package.json

```diff
@@ -0,0 +1,28 @@
+{
+    "name": "timbre",
+    "version": "14.11.25",
```

Comment:
Please do not include version, the auto-update bot will populate this once it pulls in version files.

Reply:
Updated